core/src/main/java/org/elasticsearch/common/http/client/HttpDownloadHelper.java

```
@@ -266,6 +267,14 @@ private URLConnection openConnection(URL aSource) throws IOException {
                 connection.setIfModifiedSince(timestamp);
             }
 
+            // in case the plugin manager is its own project, this can become an authenticator
+            boolean isSecureProcotol = "https".equalsIgnoreCase(aSource.getProtocol());
+            boolean isAuthInfoSet = !Strings.isNullOrEmpty(aSource.getUserInfo());
+            if (isAuthInfoSet && isSecureProcotol) {
```

should we throw an exception here too if it's set but not secure?